bin/pycbc_multi_inspiral

```diff
@@ -36,6 +36,10 @@ from pycbc.filter import MatchedFilterControl
 from pycbc.types import TimeSeries, zeros, float32, complex64
 from pycbc.types import MultiDetOptionAction
 from pycbc.vetoes import sgchisq
+# The following block of lines sets up the command-line interface (CLI) for the
+# pycbc_multi_inspiral executable. To see an example of how to feed this executable
+# to analyze the GW170817 event, take a look at:
+# https://github.com/gwastro/pycbc/blob/master/examples/multi_inspiral/run.sh
```

Comment:
Please keep two blank lines after the imports (standard PEP8 style).

I would put the URL of the example script in the docstring above.